### PR TITLE
Kusto 2020-09-18 client

### PR DESCRIPTION
**Pull request by astauben**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Kusto

**Comment by astauben** [comment]
Hi, @haroldrandom we're failing on "CLI Linter" test even though everything looks OK to me. may you please help to understand the issue?

File: src/kusto/azext_kusto/generated/_help.py [hunk 1]
```diff
@@ -1,758 +1,798 @@
...
+            Usage: --read-only-following-database hot-cache-period=XX location=XX kind=XX
+
+            hot-cache-period: The time the data should be kept in cache for fast queries in TimeSpan.
+            location: Resource location.
+            kind: Required. Kind of the database
+    examples:
+      - name: KustoDatabasesCreateOrUpdate
+        text: |-
+               az kusto database create --cluster-name "kustoclusterrptest4" --database-name "KustoDatabase8" \
+--parameters "{\\"location\\":\\"westus\\",\\"properties\\":{\\"softDeletePeriod\\":\\"P1D\\"}}" --resource-group \
```
Comment (kairu-ms) [inline src/kusto/azext_kusto/generated/_help.py:397]: @astauben --parameters is an unrecognized argument.

File: src/kusto/azext_kusto/generated/_help.py [hunk 2]
```diff
@@ -1,758 +1,798 @@
...
+            Usage: --read-only-following-database hot-cache-period=XX location=XX kind=XX
+
+            hot-cache-period: The time the data should be kept in cache for fast queries in TimeSpan.
+            location: Resource location.
+            kind: Required. Kind of the database
+    examples:
+      - name: KustoDatabasesUpdate
+        text: |-
+               az kusto database update --cluster-name "kustoclusterrptest4" --database-name "KustoDatabase8" \
+--parameters "{\\"properties\\":{\\"softDeletePeriod\\":\\"P1D\\"}}" --resource-group "kustorptest"
```
Comment (kairu-ms) [inline src/kusto/azext_kusto/generated/_help.py:427]: @astauben --parameters is an unrecognized argument.

Comment (astauben) [reply src/kusto/azext_kusto/generated/_help.py:427]: fixed. Thanks!

File: src/kusto/azext_kusto/generated/_help.py (hunk 1 above)
Comment (astauben) [reply src/kusto/azext_kusto/generated/_help.py:397]: fixed. Thanks!

**Comment by astauben** [comment]
@kairu-ms  Kind reminder  for review

**Review by haroldrandom — APPROVED** [review]

**Comment by haroldrandom** [comment]
@astauben Don't you need to upgrade the version?

**Comment by astauben** [comment]
> @astauben Don't you need to upgrade the version? The release process for Azure CLI Extension is only triggered when detecting your package version is upgraded. Please make sure of it if you want to upgrade a newer version.

Done. Thanks

File: src/kusto/HISTORY.rst [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-++++++
-* Initial release.
+.. :changelog:
+
+Release History
+===============
+
+0.1.0
+++++++
+* Initial release.
```
Comment (haroldrandom) [inline src/kusto/HISTORY.rst:8]: It would be better to leave meaningful release notes

Comment (astauben) [reply src/kusto/HISTORY.rst:8]: Done, Thanks! I'll add this to our client-generation docs for next time

**Pull request merged**